CONTRIBUTING.md

```diff
@@ -288,6 +289,11 @@ navigate to _Editor_ -> _Code Style_ -> _Java_ -> _Imports_
 and set _Class count to use import with '\*'_ to `999`.
 Do the same with _Names count to use static import with '\*'_.
 
+### GitPod
+
+You can also use [GitPod](https://gitpod.io) to contribute without install anything on your computer. Click [here](#https://gitpod.io/https://github.com/quarkusio/quarkus/-/tree/main/) to start a workspace.
```

Comment:
I'm surprised by the `#` at the beginning of the URL?

Reply:
oh the # isn't at the good location. It's https://gitpod.io/#https://github.com/quarkusio/quarkus/-/tree/main/. I will change now